.github/dependabot.yml

```diff
@@ -43,3 +43,36 @@ updates:
     interval: daily
   target-branch: "addon-resizer-release-1.8"
   open-pull-requests-limit: 3
+# Go - Cluster Autoscaler
+- directory: "/cluster-autoscaler"
+  package-ecosystem: "gomod"
+  open-pull-requests-limit: 5
+  schedule:
+    interval: "daily"
+    time: "15:00"
+    # Use America/New_York Standard Time (UTC -05:00)
+    timezone: "America/New_York"
+  commit-message:
+    prefix: "dependabot"
+    include: scope
+  ignore:
+    # Update providers manually.
+    - dependency-name: "k8s.io/cloud-provider-aws/*"
+    - dependency-name: "github.com/aws/*"
+    - dependency-name: "k8s.io/cloud-provider-gcp/*"
+    - dependency-name: "sigs.k8s.io/cloud-provider-azure/*"
+    - dependency-name: "github.com/Azure/*"
+    - dependency-name: "github.com/digitalocean/*"
+    # Maintain k8s version skew compatibility manually.
+    - dependency-name: "k8s.io/api"
```

Comment:
I wonder if we should include all the possible k/k staging modules instead of just the ones currently used (like we do with replaces in the go.mod). Otherwise if a new staging k/k module is added as CA dependency in the future, we'll treat it differently than all the ones listed here.